squirrel/commands/watch.py

```diff
@@ -72,42 +69,44 @@ def get_daemon_pid() -> int:
 
 
 def daemon(wd, logger):
-    watch_flags = flags.CREATE | flags.MODIFY | flags.DELETE
 
     os.chdir(wd)
-    logger.debug('Adding inotify watches')
-    watches = (wd, )
-    i = INotify()
-    for watch in watches:
-        i.add_watch_recursive(watch, watch_flags)
-
-    engine = plugin.load_module()
-
+    watches = wd
+    project_files = Plugin.get_files(wd)
+    logger.info(f'{Plugin.get_files.__name__}: Project files intialized.')
+    logger.info(
+        f'{Plugin.get_files.__name__}: {len(project_files)} files found')
+    engine = Plugin.load_module()
+
+    # TODO?: Add filetype to project.xml? So it can be project spesific files
+    # can be called from something like "engine.file_type"?
+    file_type = ['*.txt']
```

Comment:
Can you make it so that it watches any file type?

Reply:
Sure